controllers/webhooks/networking/cfroute_validator.go

```diff
@@ -176,12 +181,9 @@ func (v *CFRouteValidator) fetchDomain(ctx context.Context, route *korifiv1alpha
 	return domain, err
 }
 
-func (v *CFRouteValidator) validateDestinations(ctx context.Context, route *korifiv1alpha1.CFRoute) (*korifiv1alpha1.CFDomain, error) {
-	domain, err := v.fetchDomain(ctx, route)
+func (v *CFRouteValidator) validateDestinations(ctx context.Context, route *korifiv1alpha1.CFRoute) error {
+	err := v.checkDestinationsExistInNamespace(ctx, *route)
```

Comment:
Can this cause problems? Maybe not since apps remove themselves from the destinations when they're deleted?

Reply:
I think this is fine for the reason you mentioned. At the time of a route being updated, all destinations should be valid as the destination apps will still exist.